test/unit/materialized_view_activity_test.sql

```diff
@@ -0,0 +1,72 @@
+select col_hasnt_default('ggircs_swrs', 'activity', 'swrs_report_id', 'column activity.swrs_report_id should not  have a default');
+
+select has_column(       'ggircs_swrs', 'activity', 'process_name', 'column activity.process_name should exist');
+select col_type_is(      'ggircs_swrs', 'activity', 'process_name', 'character varying(1000)', 'column activity.process_name should be type text');
+select col_is_null(      'ggircs_swrs', 'activity', 'process_name', 'column activity.process_name should allow null');
+select col_hasnt_default('ggircs_swrs', 'activity', 'process_name', 'column activity.process_name should not  have a default');
+
+select has_column(       'ggircs_swrs', 'activity', 'subprocess_name', 'column activity.subprocess_name should exist');
+select col_type_is(      'ggircs_swrs', 'activity', 'subprocess_name', 'character varying(1000)', 'column activity.subprocess_name should be type text');
+select col_is_null(      'ggircs_swrs', 'activity', 'subprocess_name', 'column activity.subprocess_name should allow null');
+select col_hasnt_default('ggircs_swrs', 'activity', 'subprocess_name', 'column activity.subprocess_name should not  have a default');
+
+select has_column(       'ggircs_swrs', 'activity', 'information_requirement', 'column activity.information_requirement should exist');
+select col_type_is(      'ggircs_swrs', 'activity', 'information_requirement', 'character varying(1000)', 'column activity.information_requirement should be type text');
+select col_is_null(      'ggircs_swrs', 'activity', 'information_requirement', 'column activity.information_requirement should allow null');
+select col_hasnt_default('ggircs_swrs', 'activity', 'information_requirement', 'column activity.information_requirement should not  have a default');
+
+select has_column(       'ggircs_swrs', 'activity', 'sub_activity_xml', 'column activity.sub_activity_xml should exist');
+select col_type_is(      'ggircs_swrs', 'activity', 'sub_activity_xml', 'xml', 'column activity.sub_activity_xml should be type xml');
+select col_is_null(      'ggircs_swrs', 'activity', 'sub_activity_xml', 'column activity.sub_activity_xml should allow null');
+select col_hasnt_default('ggircs_swrs', 'activity', 'sub_activity_xml', 'column activity.sub_activity_xml should not  have a default');
+
+select has_column(       'ggircs_swrs', 'activity', 'swrs_report_history_id', 'column activity.swrs_report_history_id should exist');
+select col_type_is(      'ggircs_swrs', 'activity', 'swrs_report_history_id', 'bigint', 'column activity.swrs_report_history_id should be type bigint');
+select col_is_null(      'ggircs_swrs', 'activity', 'swrs_report_history_id', 'column activity.swrs_report_history_id should allow null');
+select col_hasnt_default('ggircs_swrs', 'activity', 'swrs_report_history_id', 'column activity.swrs_report_history_id should not  have a default');
+
```

Comment:
This file needs an import xml fixture test. There is an example in the facility unit test starting at line 43.
https://github.com/bcgov/cas-ggircs/blob/master/test/unit/materialized_view_facility_test.sql